testbed/integration/k8senrichment/README.md

```diff
@@ -0,0 +1,17 @@
+# Enrich from Kubernetes
```

Comment:
I don't love the directory structure here. It would be nice if all the utilities could maybe stay in testbed or even better, in internal, and then we can have a `tests/usecases` folder where we can place all the usecase tests we want to test.

Reply:
Alright, I will keep it for a separated PR and try to improve the structure.